### PR TITLE
Regex Task-2

### PR DESCRIPTION
**Pull request by anupm12**

### CODE REVIEW

**Review by Jigar3 — APPROVED** [review]
Good job

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "configurations": [
```
Comment (Anupam-dagar) [inline .vscode/c_cpp_properties.json]: remove .vscode directory and add to gitignore.

**Review by Anupam-dagar — CHANGES_REQUESTED** [review]
you need to remove .vscode from the commit.

File: .vscode/c_cpp_properties.json (hunk 1 above)
Comment (anupm12) [reply .vscode/c_cpp_properties.json]: I removed .vscode directory @Anupam-dagar

**Comment by anupm12** [comment]
@Anupam-dagar @Jigar3  please review now.

**Review by Anupam-dagar — APPROVED** [review]

**Pull request merged**